internal/bundlegen.js

```diff
@@ -144,7 +150,7 @@ exports.doJSBundle = function(bundle, applyImports) {
         for (var ii = 0; ii < bundle.startupModules.length; ii++) {
             var startupModule = bundle.startupModules[ii];
             if (startupModule.charAt(0) === '.') {
-                var relativeModulePath = path.relative(wrapperFileDir, startupModule);
+                var relativeModulePath = cleanPath(path.relative(wrapperFileDir, startupModule));
```

Comment:
I'm not sure this will really fix anything. I think the problem lies in `browserify`, where we need to somehow intercept and alter the paths it uses. I've experimented with using both the `fullPath` true/false options and neither one worked to properly generate bundles. We definitely need to _not_ use `fullPath`, though, as building across different systems yields different results.

Reply:
3 years later, but I can confirm these changes actually allowed us to build BlueOcean on Windows after applying the fix manually under the node_modules folders.